html2asketch/model/shapeGroup.js

```diff
@@ -4,10 +4,14 @@ class ShapeGroup extends Base {
   constructor({x, y, width, height}) {
     super();
     this._class = 'shapeGroup';
-    this._x = x;
-    this._y = y;
     this._width = width;
     this._height = height;
+    this.setPosition({x, y});
+  }
+
+  setPosition({x, y}) {
+    this._x = x;
+    this._y = y;
   }
```

Comment:
nit:  Since you extended the public API - Bitmap, Base.setHasClippingMask, ShapeGroup.setPosition - can you please update the typings? https://github.com/brainly/html-sketchapp/tree/master/typings . Sorry I didn't mention that before.

Reply:
Added! :)